### PR TITLE
[APB-1601][RV] add validations for confirm invite and terms pages

### PR DESCRIPTION
**Pull request by rvijayan**

### CODE REVIEW

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/ClientsInvitationController.scala [hunk 1]
```diff
@@ -39,19 +40,15 @@ class ClientsInvitationController @Inject() (
...
+  import ClientsInvitationController._
 
   def start(invitationId: String): Action[AnyContent] = Action.async { implicit request =>
     Future successful Ok(landing_page()).withSession(request.session + ("invitationId", invitationId))
   }
 
   def submitStart: Action[AnyContent] = Action.async { implicit request =>
     withAuthorisedAsClient { mtdItId =>
-      Future successful Redirect(routes.ClientsInvitationController.getConfirmInvitation())
+      Future.successful(Redirect(routes.ClientsInvitationController.getConfirmInvitation()))
```
Comment (Smudgey) [inline app/uk/gov/hmrc/agentinvitationsfrontend/controllers/ClientsInvitationController.scala]: Did this need to be changed? :stuck_out_tongue:

File: conf/messages [hunk 2]
```diff
@@ -78,6 +78,9 @@ client-complete.sub-header=What happens next
 client-complete.p1=You can manage your relationships with your agent(s) from within your Business Tax account. This account gives you direct access to information from HMRC.
 client-complete.button=Continue to your Business Tax account
 
+error.confirmInvite.invalid=Please choose an on option below
```
Comment (Smudgey) [inline conf/messages]: Is this text correct?

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/ClientsInvitationController.scala (hunk 1 above)
Comment (rvijayan) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/ClientsInvitationController.scala]: Not really, I added a validation there which was supposed to be done in another ticket, but didn't revert back fully.

File: conf/messages (hunk 2 above)
Comment (rvijayan) [reply conf/messages]: Good catch! :D

File: conf/application.conf [hunk 3]
```diff
@@ -44,7 +44,7 @@ microservice {
             host = graphite
             port = 2003
             prefix = play.${appName}.
-            enabled = true
+            enabled = false
```
Comment (Smudgey) [inline conf/application.conf]: Should this have been changed to false now?

Comment (rvijayan) [reply conf/application.conf]: This is for avoiding the exceptions thrown while running the app. We can enable graphite in app configs.

**Pull request merged**